ios/Podfile.lock

```diff
@@ -14,6 +14,21 @@ PODS:
   - boost-for-react-native (1.63.0)
   - CocoaAsyncSocket (7.6.5)
   - DoubleConversion (1.1.6)
+  - EXConstants (10.1.3):
```

Comment:
I think maybe we need to re-install iOS pods after the change I suggested. There are a lot of modules being added still.